app/models/remote_authenticatable.rb

```diff
@@ -0,0 +1,119 @@
+module Devise
+  module Models
+    module RemoteAuthenticatable
```

Comment:
Could we name this to include Cognito as a name?

Reply:
No as this extend/overrides specific functionality in Devise.  Originally this was called CognitoAuthenticatable but devise wouldn't load in properly.